### DISCUSSION TITLE
ShadButton: Rename text to widget?

### DISCUSSION THREAD

**Discussion by Ladegeraet** [nank1ro/flutter-shadcn-ui#89] [Ideas]
Hey, I think it would be nice to rename the text property of the ShadButton to child. This feels more intuitive to me (Flutter devs?) and is more similar to the original Flutter API. When I first implemented the Button, I had to look into the documentation :-p (I would figure this out without the documentation, but I think you know what I mean.)

By the way, great job on the package! It's really nice!

**Comment by nank1ro** [comment]
Hi, thanks for the suggestion.
is something I've thought about as well.
The difference with Material's ElevatedButton is that the ShadButton has both `text` where you enter a text, and `icon` where you enter an image.
That is why I preferred to differentiate it.
Instead with the elevated button you would use child for both using a Row. Or the `ElevatedButton.icon` factory constructor.

**Comment by Ladegeraet** [comment]
Resolved in Version: [v0.7.1](https://github.com/nank1ro/flutter-shadcn-ui/releases/tag/v0.7.1 )
